src/main/java/frc/team2412/robot/commands/launcher/LaunchCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class LaunchCommand extends Command {
+	private final LauncherSubsystem launcherSubsystem;
+	private final IntakeSubsystem intakeSubsystem;
+
+	public LaunchCommand(LauncherSubsystem launcherSubsystem, IntakeSubsystem IntakeSubsystem) {
```

Comment:
nit: make the `IntakeSubsystem` variable camelcase.

Reply:
done